app/controllers/schools/manual_readings_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Schools
+  class ManualReadingsController < ApplicationController
+    load_and_authorize_resource :school
+    load_and_authorize_resource through: :school
+
+    before_action :set_breadcrumbs
+
+    def show
+      @meter_start_dates = %i[electricity gas].select { |fuel_type| @school.configuration.fuel_type?(fuel_type) }
+                                              .index_with do |fuel_type|
+        @school.configuration.meter_start_date(fuel_type)
+      end
+      return if @meter_start_dates.empty?
```

Comment:
Presume this will only return here if there are no fuel types configured for the school. Does this behave when rendering the page? Assume it give the "your school has enough data message" which wouldn't quite be right. Depends if it is possible for this to happen I guess.